File: Assignment3/10.py
# Initialize counters
pos = neg = even = odd = 0

print("Enter 25 numbers:")
numbers = [int(input()) for _ in range(25)]

for num in numbers:
    if num > 0:
        pos += 1
    elif num < 0:
        neg += 1

    if num % 2 == 0:
        even += 1
    else:
        odd += 1

print("Positive numbers:", pos)
print("Negative numbers:", neg)
print("Even numbers:", even)
print("Odd numbers:", odd)

File: Assignment3/10.c
#include <stdio.h>

int main() {
    int arr[25];
    int pos = 0, neg = 0, even = 0, odd = 0;

    printf("Enter 25 numbers:\n");
    for (int i = 0; i < 25; i++) {
        scanf("%d", &arr[i]);

        if (arr[i] > 0)
            pos++;
        else if (arr[i] < 0)
            neg++;

        if (arr[i] % 2 == 0)
            even++;
        else
            odd++;
    }

    printf("Positive numbers: %d\n", pos);
    printf("Negative numbers: %d\n", neg);
    printf("Even numbers: %d\n", even);
    printf("Odd numbers: %d\n", odd);

    return 0;
}
